server/router_utils.go

```diff
@@ -196,3 +196,30 @@ func (server *HTTPServer) readClusterRuleUserParams(
 
 	return clusterID, ruleID, userID, true
 }
+
+// readClusterRuleParams gets cluster_name and rule_id from current request
+func (server *HTTPServer) readClusterRuleParams(
+	writer http.ResponseWriter, request *http.Request,
+) (types.ClusterName, types.RuleID, bool) {
```

Comment:
Nitpick: As you are returning 3 values here, probably using named return values will improve the readability.

Reply:
definitely a good idea to do that, we have a lot of such functions in aggregator, even worse than this one. Might be a good candidate for some maintenance/readability tasks.